src/commands/wizard/lwcGenerationCommand.ts

```diff
@@ -8,6 +18,70 @@ export class LwcGenerationCommand {
         this.extensionUri = extensionUri;
     }
 
+    static readFileAsJsonObject(
+        filePath: string,
+        callback: (error: Error | null, data: any) => void
+    ): void {
+        try {
+            const data = fs.readFileSync(filePath, { encoding: 'utf-8' });
+            const jsonObject = JSON.parse(data);
+            callback(null, jsonObject);
+        } catch (error) {
+            callback(error as Error, null);
+        }
+    }
+
+    static async getLwcGenerationPageStatus(): Promise<LwcGenerationCommandStatus> {
+        return new Promise<LwcGenerationCommandStatus>(async (resolve) => {
+            let landingPageExists = true;
+
+            const staticResourcesPath =
+                await TemplateChooserCommand.getStaticResourcesDir();
```

Comment:
@dbreese and I (coincidentally) discussed this earlier: these commands should move out into independent helper functionality. I'd prefer we do that, over importing `TemplateChooserCommand` to get to them.

Reply:
I see this is resolved, but...can we do this? 😀 Is that TBD?